spec/requests/user/_profile_header_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
```

Comment:
I don't think this file is needed as there's already a `user_show_spec.rb`. After all you're testing the page, not the view partial